src/storage/mod.rs

```diff
@@ -1446,6 +1544,45 @@ impl<E: Engine, L: LockMgr> Storage<E, L> {
             .flatten()
     }
 
+    /// Get the values of a set of raw keys, return a vector of results.
+    pub fn batch_async_raw_get(
```

Comment:
This name is easy to confuse with the interface `async_raw_batch_get `.
Maybe we could rename it to `async_raw_batch_get_command` and rename `async_raw_batch_get ` to ``async_raw_batch_get_key`

Reply:
Sounds nice.